src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -300,11 +300,11 @@ void M1()
   Instance Receiver: IInstanceReferenceExpression (InstanceReferenceKind.Implicit) (OperationKind.InstanceReferenceExpression, Type: P) (Syntax: 'M2')
   Arguments(2):
       IArgument (ArgumentKind.Explicit, Matching Parameter: x) (OperationKind.Argument) (Syntax: 'a')
-        ILocalReferenceExpression: a (OperationKind.LocalReferenceExpression, Type: System.Int32) (Syntax: 'a')
+        ILocalReferenceExpression: a (IsDeclaration: False) (OperationKind.LocalReferenceExpression, Type: System.Int32) (Syntax: 'a')
```

Comment:
I feel like we could avoid a lot of test churn, and make it easier to grok relevant info, if we elided the common values.  i.e. only put this is if "IsDeclaration: true"

Reply:
*Or*.  If we do want to always include this value, do the PR where you don't add the "IsDeclaratin: false" cases.  Have us review and submit.  Then make a test only change that adds the value back in for all cases.  Then the second is mechanical and easy to review.